components/widgets/menu/menu.test.jsx

```diff
@@ -111,4 +111,102 @@ describe('components/Menu', () => {
 </ul>
 `);
     });
+
+    test('should hide the correct dividers', () => {
+        const utils = require('utils/utils'); //eslint-disable-line global-require
+        utils.isMobile.mockReturnValue(false);
+        const pseudoMenu = document.createElement('div');
+        const listOfItems = [
+            'menu-divider', 'menu-divider', 'menu-divider', 'menu-divider',
+            'other', 'other',
+            'menu-divider', 'menu-divider',
+            'other',
+            'menu-divider', 'menu-divider', 'menu-divider',
+            'other', 'other', 'other',
+            'menu-divider', 'menu-divider', 'menu-divider', 'menu-divider',
```

Comment:
Can we have another test for `mobile-menu-divider`?

Reply:
sure

Reply:
done